docs/03-tutorials/00-api-exposure/apix-02-setup-custom-domain-for-workload.md

```diff
@@ -1,71 +1,63 @@
 
    ```bash
-   export NAMESPACE={NAMESPACE_NAME}
-   kubectl create ns $NAMESPACE
-   kubectl label namespace $NAMESPACE istio-injection=enabled --overwrite
+   export SECRET={SECRET_NAME}
    ```
 
-2. Create a Secret containing credentials for your DNS cloud service provider account in your Namespace.
-
-  See the [official External DNS Management documentation](https://github.com/gardener/external-dns-management/blob/master/README.md#external-dns-management), choose your DNS cloud service provider, and follow the relevant guidelines. Then run:
-
-  ```bash
-  kubectl apply -n {NAMESPACE_NAME} -f {SECRET}.yaml
-  ```
-
-3. Create a DNSProvider and a DNSEntry custom resource (CR).
+2. Create a DNSProvider and a DNSEntry custom resource (CR).
 
    - Export the following values as environment variables and run the command provided.
   
    As the **SPEC_TYPE**, use the relevant provider type. See the [official Gardener examples](https://github.com/gardener/external-dns-management/tree/master/examples) of the DNSProvider CR.
 
    ```bash
    export SPEC_TYPE={PROVIDER_TYPE}
-   export SECRET={SECRET_NAME}
-   export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME} # The domain that you own, e.g. mydomain.com.
+   export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME} 
    ```
+   >**NOTE:** `DOMAIN_NAME` is the domain that you own, for example, mydomain.com
 
-    ```bash
-    cat <<EOF | kubectl apply -f -
-    apiVersion: dns.gardener.cloud/v1alpha1
-    kind: DNSProvider
-    metadata:
-      name: dns-provider
-      namespace: $NAMESPACE
-      annotations:
-        dns.gardener.cloud/class: garden
-    spec:
-      type: $SPEC_TYPE
-      secretRef:
-        name: $SECRET
-      domains:
-        include:
-          - $DOMAIN_TO_EXPOSE_WORKLOADS
-    EOF
-    ```
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: dns.gardener.cloud/v1alpha1
+   kind: DNSProvider
+   metadata:
+     name: dns-provider
+     namespace: $NAMESPACE
+     annotations:
+       dns.gardener.cloud/class: garden
+   spec:
+     type: $SPEC_TYPE
+     secretRef:
+       name: $SECRET
+     domains:
+       include:
+         - $DOMAIN_TO_EXPOSE_WORKLOADS
+   EOF
+   ```
 
    - Export the following values as environment variables and run the command provided:
 
    ```bash
-   export WILDCARD={WILDCRAD_SUBDOMAIN} #e.g. *.api.mydomain.com
-   export IP=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}') # assuming only one LoadBalancer with external IP
+   export IP=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}') # Assuming only one LoadBalancer with external IP
```

Comment:
Can you please extend here the comment or add is as a note that for getting the IP:
`Sometimes you may need to use jsonpath='{.status.loadBalancer.ingress[0].hostname}' instead of ip, depending on cluster provider, for example on AWS`